scripts/verify_ai_config_flow.py

```
@@ -0,0 +1,156 @@
+state.update(disk=dict(DEFAULT, enabled=True, apiKey=encrypt_cpp("sk-keep-me", MID, HOME)),
+             loaded=False, cache=None)
+load_config()
+save_config({"apiKey": "sk-****1234"})
+check("掩码不覆盖原 key", state["cache"]["apiKey"] == "sk-keep-me")
+
+print("== 场景7: 密文损坏/换机器 → 安全清空 ==")
+state.update(disk=dict(DEFAULT, enabled=True, apiKey="enc:v1:Zm9vYmFy" * 5), loaded=False, cache=None)
+cfg = load_config()
+check("解密失败后 key 清空", cfg["apiKey"] == "")
+check("进程不崩溃、继续返回配置", cfg["enabled"] is True)
+check("记录错误日志", any("decrypt-fail" in l for l in state["logs"]))
```

Flake8 (7.3.0)

[error] ambiguous variable name 'l'

(E741)